### DISCUSSION TITLE
Welcome to DeepCalib Discussions!

### DISCUSSION THREAD

**Discussion by alexvbogdan** [alexvbogdan/DeepCalib#16] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GGbugmanTED** [comment]
Thanks for the sharing. May I ask something about error calculation?
For classification only, the predicted result is the label of focal/distortion. How do we calculate the error?
Is it  Label value - ground-truth value,   then average all the test results for the image? 
Also, in paper table 1: f, distortion, how to get these number, for example, f=954, xi = 0.77

**Reply by GGbugmanTED** [reply]
if not converted into mm unit, the value should be the label category value

**Reply by GGbugmanTED** [reply]
I saw the SUPPLEMENTARY paper. it is said "... In this paper, the distortion error stands for the absolute difference between the predicted distortion parameter and its ground-truth value. "

**Comment by NasrulMubin** [comment]
Hello, I've tried to reproduce this. But, actually I've got stuck. I think there are some files that isn't available. I've got stuck at the "training". What is the "valid/*.jpg" consist of? I mean which image is this belong to? Please give me a clue. Looking forward for it. Thanks
